akka-http-caching/src/main/scala/akka/http/caching/scaladsl/Cache.scala

```diff
@@ -39,14 +40,16 @@ abstract class Cache[K, V] extends akka.http.caching.javadsl.Cache[K, V] {
   /**
    * Returns either the cached Future for the given key or the given value as a Future
    */
-  def apply(key: K, block: () ⇒ V): Future[V] =
+  def get(key: K, block: () ⇒ V): Future[V] =
```

Comment:
this cannot be `apply` because it clashes with `def apply(key: K, genValue: () ⇒ Future[V]): Future[V]` after type erasure